### PR TITLE
Fix for embedding InputURI.

### LINKED ISSUE #1515: field of type resource with no referenced RT does not work when embedded in another RT
on development, the template test:resource:noRT has one field, which is type = resource, with no other resource templates specified, therefore expects a URI. This template works fine, but when embedded in another template:

test:embed:resource:noRT, when you try to add the field, it kicks you to a blank screen.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1515

### CODE REVIEW

**Comment by justinlittman** [comment]
This can be tested by adding the 2 attached templates to `__tests__/__fixtures__` and adding to `fixtureLoaderHelper`.

```
{
  "propertyTemplates": [
    {
      "mandatory": "false",
      "repeatable": "true",
      "type": "resource",
      "resourceTemplates": [],
      "valueConstraint": {
        "valueTemplateRefs": [],
        "useValuesFrom": [],
        "defaults": [],
        "valueDataType": {}
      },
      "propertyURI": "http://id.loc.gov/ontologies/bibframe/genreForm",
      "propertyLabel": "this field expects a URI"
    }
  ],
  "id": "test:resource:noRT",
  "resourceLabel": "I have one field",
  "resourceURI": "http://id.loc.gov/ontologies/bibframe/Work",
  "author": "michelle",
  "schema": "https://ld4p.github.io/sinopia/schemas/0.0.9/resource-template.json"
}
```


```
{
  "propertyTemplates": [
    {
      "mandatory": "false",
      "repeatable": "true",
      "type": "resource",
      "resourceTemplates": [],
      "valueConstraint": {
        "valueTemplateRefs": [
          "test:resource:noRT"
        ],
        "useValuesFrom": [],
        "defaults": [],
        "valueDataType": {}
      },
      "propertyURI": "http://id.loc.gov/ontologies/bibframe/date",
      "propertyLabel": "i embed the test:resource:noRT template"
    }
  ],
  "resourceURI": "http://id.loc.gov/ontologies/bibframe/Work",
  "id": "test:embed:resource:noRT",
  "resourceLabel": "I embed one resource template",
  "author": "michelle",
  "schema": "https://ld4p.github.io/sinopia/schemas/0.0.9/resource-template.json"
}
```

File: src/components/editor/property/InputURI.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const InputURI = (props) => {
     return errors
   }
 
-
+  console.log('items', props.items)
```
Comment (jmartin-sul) [inline src/components/editor/property/InputURI.jsx]: stray debug statement?  (will test out locally per instructions and leave an actual review in a bit)

**Comment by justinlittman** [comment]
Yes @jcoyne but will require re-writing InputURI tests with RTL. This isn't testable with current enzyme tests.

**Comment by justinlittman** [comment]
@jcoyne The first test would fail without the fix.

**Comment by justinlittman** [comment]
@jcoyne Commit message improved.

**Pull request merged**